src/xbot/common/command/BaseSubsystem.java

```diff
@@ -13,5 +14,10 @@ public void setDefaultCommand(Command command) {
     protected void initDefaultCommand() {
 
     }
+    
+    public void updateTelemetry() {
```

Comment:
That's fair, I was debating between the two options and had leaned this way to reduce the number of things people had to learn about (like another interface they need to add to their subsystem). That said, the non-subsystem class case seems like a reasonable one.